app/kumactl/cmd/install/testdata/install-logging.defaults.golden.yaml

```diff
@@ -687,4 +687,4 @@ spec:
           secret:
             secretName: loki
         - name: storage
-          emptyDir: {}
+          emptyDir: {}
```

Comment:
Does it mean a new update mode for golden files isn't adding an empty line at the end?

Reply:
maybe. It's just overriding file with the output content. If content has no empty line at the end, there won't be empty line in the golden file. Is that a problem?

Reply:
No, no problem, GitHub so much highlights the fact there is no empty line at the end, so I asked :)